ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -582,7 +582,18 @@ def verify_storage_system():
     managed_service = (
         config.ENV_DATA["platform"].lower() in constants.MANAGED_SERVICE_PLATFORMS
     )
+    live_deployment = config.DEPLOYMENT.get("live_deployment")
+    ocp_version = version.get_semantic_ocp_version_from_config()
     ocs_version = version.get_semantic_ocs_version_from_config()
+    if (
+        live_deployment
+        and ocs_version == version.VERSION_4_9
+        and ocp_version == version.VERSION_4_10
+    ):
+        log.warning(
+            "Because of the BZ 2079878, we are skipping storage system validation!"
```

Comment:
BZ 2079878 is for ODF 4.10 upgrade ( from 4.10 live to internal build ). above validation is for ODF 4.9

Reply:
Yes, my mistake when copy paste from wrong tab :). I will update to: 2075422. Thanks